app/unite-framework/unitemenu.ts

```diff
@@ -1,26 +1,48 @@
+import {LocalStorage,Storage} from 'ionic-angular';
 import {Injectable, Inject} from 'angular2/core';
-import {Http} from 'angular2/http';
-import 'rxjs/add/operator/map';
-// import {GettingStartedPage} from '../pages/getting-started/getting-started';
-// import {ListPage} from '../pages/list/list';
-
-
 @Injectable()
 export class UniteMenu {
   pages: any;
```

Comment:
@coolbung Need to change pages object node dynamically. So i kept it same (pages: any)